papers/system-f-in-agda/errata.txt

```diff
@@ -0,0 +1,7 @@
+This file documents bugs/fixes made after publication:
```

Comment:
Should we include this as a section in the paper?

Reply:
I think it would clutter the paper and isn't very useful to the reader. I would prefer to keep it separate.